R/add_subprotocol.R

```diff
@@ -48,18 +50,9 @@ add_subprotocol <-
   function(code_subprotocol,
            version_number,
            file_name,
-           section,
-           demote_header = 0,
-           params) {
-
-    execshell <- function(commandstring, intern = FALSE) {
-      if (.Platform$OS.type == "windows") {
-        res <- shell(commandstring, intern = TRUE)
-      } else {
-        res <- system(commandstring, intern = TRUE)
-      }
-      if (!intern) cat(res, sep = "\n") else return(res)
-    }
+           section = NULL,
+           demote_header = c(0, 1, 2, -1),
```

Comment:
This is what I did initially, but I had to change it afterwards because `match.arg(demote_header)` errors when non-character values are used (because it uses `pmatch()` to match choices).
Can you revert line 54?

Reply:
> `match.arg(demote_header)` errors when non-character values are used (because it uses `pmatch()` to match choices)

Interesting, didn't know.

Reply:
Discussion continued below in https://github.com/inbo/protocolhelper/pull/39#discussion_r574502898